src/components/Text/__stories__/Variants.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { ComponentStory } from '@storybook/react'
+import Text, { textVariants } from '../index'
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const Variants: ComponentStory<typeof Text> = props => (
```

Comment:
Pass props to `Text` and remove the eslint disable :)

Reply:
@baptisteArno I cannot in this cas I will have errors on properties I passing to Text

![Screenshot 2022-10-17 at 09 57 39](https://user-images.githubusercontent.com/15812968/196121165-d1e60126-c0c2-48b8-aa34-f69694b656d9.png)

Reply:
Does it work like this?

```
<Text as="div" key={variant} {...props} variant={variant}>
  {variant}
</Text>
```

Reply:
@baptisteArno Yes it does!